android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/components/MeasureReportPopulationResultView.kt

```diff
@@ -60,7 +60,7 @@ const val POPULATION_REPORT_INDIVIDUAL_RESULT_COUNT_TEST_TAG =
 
 @Composable
 fun MeasureReportPopulationResultView(dataList: List<MeasureReportPopulationResult>) {
-  LazyColumn { items(dataList, key = { it.title }) { item -> PopulationResultCard(item) } }
+  LazyColumn { itemsIndexed(dataList) { _, item -> PopulationResultCard(item) } }
```

Comment:
Do we expect to use the indices for the items in future? If not then I think `items(...){}` will do just fine? Also `LazyColumn` requires a unique key to identify the items in the list to avoid side-effects that may occur during re-composition, it's recommended to explicitly declare the identifier.

Reply:
@ellykits  Previously, We were generating reports for a single indicator , and the title was unique lets say  by month report will be generated single time. but in this case we have multiple indicators now which can have duplication of title.  We have 2 solutions for this
->Add unique id in the data class for each MeasureReportPopulationResult 
->Make use of itemsIndexed which does not require a unique id.